Writerside/topics/Quickstart-Up-And-Running.md

```diff
@@ -0,0 +1,528 @@
+# Quickstart: Running Your Game With Catena
+
+In this quickstart guide we will walk through how to set up and run your game on your own infrastructure using Catena Tools. 
+Through this guide you will set up a backend, basic fleet manager, along with a game client and game server. This is a __Static Fleet Management Solution.__ 
+This means you will have to manually provision servers, but they will report themselves to the backend.
+
+> This tutorial will use the following Catena Components to support a Multiplayer Workflow.
+>   * [Match Broker](Match-Broker.md)
+>   * [Server Manager](Server-Manager.md)
+>   * [Sidecar](Sidecar.md)
+>   * [Matchmaking](Matchmaking.md)
+>   * Infrastructure Templates
+
+## Overview
+
+By the end of this quickstart, you will be able to run your game, or a sample game in a multiplayer/online environment using AWS. Players will be able to:
+1. Log into your game
+2. Form a Party
+3. Request a Match
+4. Connect to a Server
+5. Play a game
+
+## Prerequisites
+
+What you will need before following the steps in this guide.
+
+* Access to Catena Tools
+* A Game with a client server model or the [Catena Lyra Demo](https://github.com/catenaTools/catena-lyra-demo)
+* The Catena Tools [AWS Infrastructure Template](https://github.com/CatenaTools/infrastructure/tree/main/aws/catena-core)
+* [Terraform](https://www.terraform.io) V1.71 or higher
+* An AWS account
+* Git bash (optional)
+* A domain name that you control and can configure in route53
+
+## Setup the Infrastructure
+
+Prior to running the Terraform code, set up a hosted zone for the domain that you will use. In the example, this will be `catenatools.com.`
```

Comment:
Prior to provisioning any infrastructure, you will need to set up a hosted zone in Route 53 for the domain that you will use.